godtools/App/Share/Domain/UseCases/GetToolUseCase/ToolDomainModel.swift

```diff
@@ -10,9 +10,10 @@ import Foundation
 
 struct ToolDomainModel {
     
-    // TODO: - finish mapping necessary attributes in GT-1777
+    let abbreviation: String
     let bannerImageId: String
     let category: String
+    let currentTranslationLanguage: LanguageDomainModel?
```

Comment:
Hey @rachaelblue I don't think this needs to be optional now does it?

Reply:
nope, you're right